Header Header

float64 starter_consistency

float64 time_stamp
float64 ender_consistency
float64 counter
float64 check_sum

uint16 ITC_info
bool hardware_fail
bool SGU_fail
float64 horizontal_misalign
float64 absorb_blind
float64 distort_blind
uint8 message_counter
uint8 message_CRC
